boards/st/nucleo_g474re/doc/index.rst

```diff
@@ -207,6 +207,17 @@ Flashing
 
 Nucleo G474RE board includes an ST-LINK/V3E embedded debug tool interface.
 
+The board is configured to be flashed using west `STM32CubeProgrammer`_ runner,
+so its installation is required.
+
+Alternatively, openocd or JLink can also be used to flash the board using
+the ``--runner`` (or ``-r``) option:
+
+.. code-block:: console
+
+   $ west flash --runner openocd
+   $ west flash --runner jlink
```

Comment:
Additionally to stm32cubeprogrammer, it only has pyocd and openocd.
But jlink could be added in a follow up PR.